packages/meditrak-server/src/routes/GETHandler/helpers.js

```diff
@@ -83,6 +83,7 @@ export const getQueryOptionsForColumns = (
   columnNames,
   baseRecordType,
   customJoinConditions = {},
+  defaultJoinType = null,
```

Comment:
```suggestion
  joinType = null,
```
At this point it's probably just the join type